codex-rs/app-server/src/codex_message_processor.rs

```diff
@@ -948,25 +1583,38 @@ impl CodexMessageProcessor {
             None => None,
         };
 
-        let response = ListConversationsResponse { items, next_cursor };
-        self.outgoing.send_response(request_id, response).await;
+        Ok((items, next_cursor))
     }
 
-    async fn list_models(&self, request_id: RequestId, params: ListModelsParams) {
-        let ListModelsParams { page_size, cursor } = params;
```

Comment:
(GitHub is not letting me comment on the desired line...)

Why would the caller ever want the models in reverse order? Do we need to expose this?